services/ordering/src/EShopOnAbp.OrderingService.Application.Contracts/OrderingServiceRemoteServiceConsts.cs

```diff
@@ -1,7 +1,8 @@
 ﻿namespace EShopOnAbp.OrderingService
 {
-    public class OrderingServiceRemoteServiceConsts
+    public static class OrderingServiceRemoteServiceConsts
     {
         public const string RemoteServiceName = "Ordering";
+        public const int Top10 = 10;
```

Comment:
This is the wrong layer. Constants should be under Domain.Shared especially when you are planning to use them on EntityFrameworkCore layer.